tests/setup-softhsm.sh

```diff
@@ -137,10 +137,14 @@ ca_sign() {
     CN=$3
     KEYID=$4
     let "SERIAL+=1"
+    # BSD sed expects a space between -i and the backup extension, GNU sed
+    # expects the extension right after the -i argument. Portable scripts thus
+    # can't use sed -i.
     sed -e "s|cn = .*|cn = $CN|g" \
         -e "s|serial = .*|serial = $SERIAL|g" \
         -e "/^ca$/d" \
-        -i ${TMPPDIR}/cert.cfg
```

Comment:
can you use --in-place=suffix ? or is that not supported on bsd either?

Reply:
Not supported either.

Reply:
sigh

Reply:
I think this is broken also in the tool to generate code, but that is not require for building, so let's leave that one alone.